main.go

```diff
@@ -59,6 +45,41 @@ func update() {
 	fmt.Println("Updated Successfully.")
 }
 
+// Can be called with jsonfile=test (for example if you have a test.json file)
+// TODO: Make accepting "jsonfile=test.json" possible
+func parseJSON(jsonFilePath string) {
+	//	Open the provided file
+	jsonFile, err := os.Open(jsonFilePath)
+	if err != nil {
+		fmt.Println("File Not Found")
+		os.Exit(1)
```

Comment:
Its generally not good practice to exit from within a function, the error should be propagated back to the caller and handled there. No need to change the code now, but once you add more functionalities to the code, you would need a better error handling mechanism. Consider reading this for more examples - https://medium.com/rungo/error-handling-in-go-f0125de052f0